lib/views/Page404.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var React = require('react');
+
+module.exports = React.createClass({
+
+  render: function render() {
+
+    return React.createElement(
+      'html',
+      null,
+      React.createElement(
+        'head',
+        null,
+        React.createElement('meta', {
+          charSet: 'utf-8'
+        })
+      ),
+      React.createElement(
+        'body',
+        null,
+        React.createElement(
```

Comment:
it will be good to let the user know which url resulted in this 404.
Example: https://github.com/krakenjs/kraken-js/blob/36df428533aea2c427063901fd33cbbedb5ac324/middleware/404.js#L26